fastscanner/tests/test_eodhd_fundamental_store.py

```diff
@@ -0,0 +1,118 @@
+import os
```

Comment:
When running the tests, files get created in the data folder. We don't want to create data every time we run the tests.
You can create a fixture that returns a [TemporaryDirectory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) to avoid this.